packages/apps/project/src/components/pages/ReportingView/ReportingView.tsx

```diff
@@ -48,26 +57,26 @@ const ReportingView = () => {
         fixedItems={
           <>
             <ProjectHeader />
-            {project?.isBusinessProject && (
+            {project?.isBusinessProject && showReporting && showActivities && (
```

Comment:
As business project is already checked in hook function, you can only check showReporting && showActivities